extensions/interactions/ItemSelectionInput/directives/ItemSelectionInputValidationService.js

```diff
@@ -177,8 +180,31 @@ oppia.factory('ItemSelectionInputValidationService', [
                       'rule must include at least 2 options.')
                   });
                 }
+              } else if (rule.type === 'Equals') {
+                if (minAllowedCount > ruleInputs.length) {
+                  warningsList.push({
+                    type: WARNING_TYPES.ERROR,
+                    message: (
+                      'In answer group ' + (answerIndex + 1) + ', ' +
+                      'rule ' + (ruleIndex + 1) + ', the "Equals" rule ' +
+                      'should not have more correct options than minimum ' +
+                      'allowed selection count.')
+                  });
+                }
               }
             });
+            if (ruleInputs.length === 0) {
+              if (rule.type === 'ContainsAtLeastOneOf') {
+                warningsList.push({
+                  type: WARNING_TYPES.ERROR,
+                  message: (
+                    'In answer group ' + (answerIndex + 1) + ', ' +
+                    'rule ' + (ruleIndex + 1) + ', the ' +
+                    '"ContainsAtLeastOneOf" rule should not have more ' +
+                    'correct options than minimum allowed selection count.')
```

Comment:
Ditto. Update the error msg accordingly.

Reply:
Yeah I will do it

Reply:
Hi @amey-kudari, please follow this process when replying to comments:
- Start working on a commit to the PR to fix it
- As you fix each item, leave a reply comment, but click "Start a review" instead (or, after that, whatever button lets you add a comment to an existing set of comments). Try to leave a definitive comment like "Done" rather than "Will do".
- Push your changes.
- Once you have verified that your changes are correct on GitHub, submit your "review" with all the replies and add a top-level message saying something like `Addressed all comments, @XXX PTAL`.

Thanks!

Reply:
Hi @seanlip 
 Sorry i did not know it, I will follow the process from now. 
Thanks!

Reply:
No problem. Thanks!

Reply:
Done, updated

Reply:
This error message is still wrong. Have you understood this part of the code you've written? It raises warning only when there is no selected choice (i.e. at least one choice should be selected).